applications/RANSApplication/python_scripts/formulations/formulation.py

```diff
@@ -0,0 +1,177 @@
+import KratosMultiphysics as Kratos
+
+class Formulation:
```

Comment:
```suggestion
class RansFormulation:
```
as it can be easily misunderstood once we merge the RANS and the FluidDynamicsApplications.